www/hockeyapp.js

```diff
@@ -20,6 +20,13 @@ var hockeyapp = {
     feedback: function (success, failure) {
         exec(success, failure, "HockeyApp", "feedback", []);
     },
+    composeFeedback: function (success, failure, attachScreenshot, data) {
+        var parameters = [attachScreenshot === true || attachScreenshot === "true"];
+        if (data != undefined) {
+            parameters.push(JSON.stringify(data));
```

Comment:
That's an interesting suggestion. The downside is that the generated file may not always be possible to parse as JSON. If `data `is a string and passed through as-is it will be missing the surrounding quotes and may contain invalid sequences of characters. I suppose it's ok to leave it up to the caller to run `JSON.stringify` on the string if that is an issue.